portfolio/src/main/webapp/index.html

```diff
@@ -58,16 +60,30 @@ <h2> Fun Facts</h2>
     </p>
     
     <div id="pets"> 
-        <img src="images/Blackbeak.jpg">
-        <img src="images/Max.jpeg"> 
+      <img src="images/Blackbeak.jpg">
+      <img src="images/Max.jpeg"> 
     </div>
     
     <h3>Click Here To Get A Randomized Song Recommendation From Me!</h3>
-    
     <button onclick="addRandomSong()">Click Here!</button>
-    
     <div id="song-container"></div>
-    
+
+    <h3>Click Here For More Random Facts About Me!</h3>
+    <button onclick="moreFacts()"> More Fun facts</button>
+    <p id="facts-container"></p>  
+      
+      <form action="/form-handler" method="POST">
+        
+        <p> <b>Write Any Feedback Down Below:</b></p>
```

Comment:
You can add a class to the `<p>` tag and use `font-weight` in CSS to make this bold rather than using markup (`<b>` tag) to style your content.